src/chrome/content/rules/Tandlakarforbundet.se.xml

```diff
@@ -1,7 +1,3 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused')
-Fetch error: http://www.tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused') tandlakarforbundet is a swedish labour union -->
 <ruleset name="Tandlakarforbundet.se">
```

Comment:
This is not a bug but an edge case for _https-everywhere-checker_.  (see [`check_rules.py#L218`](https://github.com/EFForg/https-everywhere/blob/master/test/rules/src/https_everywhere_checker/check_rules.py#L218)) 

Since the original comment did not start with a newline (923fd05e), neither do _https-everywhere-checker_ suffixed its message with newline `\n\n` (b69a6d7a in 2015). The original comment and checker's message ended up in the same line. 

Since this change does not matter here I am not going to fix it. I will pay more attention to ruleset like this in later PR. 

Thanks!!